File: frontend-android/clases/Usuario.kt
package com.example.myapplication

import java.time.LocalDateTime

data class Usuario(
    var usuarioId: Int,
    var contraseña: String,
    var admin: Boolean,
    var nombreUsuario: String,
    var nombre: String,
    var apellidos: String,
    var correo: String,
    var imagen: String,
    var descripcion: String,
    var edad: LocalDateTime,
    var comunidades: MutableList<Comunidad> = mutableListOf(),
    var puntos: Int,
    var puntosExperiencia: Int,
    var nivel: Nivel,
    var productos: MutableList<Producto> = mutableListOf()
) {
    constructor() : this(
        usuarioId = 0,
        contraseña = "",
        admin = false,
        nombreUsuario = "",
        nombre = "",
        apellidos = "",
        correo = "",
        imagen = "",
        descripcion = "",
        edad = LocalDateTime.now(),
        puntos = 0,
        puntosExperiencia = 0,
        nivel = Nivel(),
        comunidades = mutableListOf(),
        productos = mutableListOf()
    )
}

File: frontend-wpf/C#Clases/Usuario.cs
public class Usuario
    {
        public int UsuarioId { get; set; }
        public string Contraseña { get; set; }
        public bool Admin { get; set; }
        public string NombreUsuario { get; set; }
        public string Nombre { get; set; }
        public string Apellidos { get; set; }
        public string Correo { get; set; }
        public string Imagen { get; set; }
        public string Descripcion { get; set; }
        public DateTime Edad { get; set; }
        public List<Comunidad> Comunidades { get; set; }
        public int Puntos { get; set; }
        public int PuntosExperiencia { get; set; }
        public Nivel Nivel { get; set; }
        public List<Producto> Productos { get; set; }

        public Usuario()
        {
            Comunidades = new List<Comunidad>();
            Productos = new List<Producto>();
        }

        public Usuario(int usuarioId, string contraseña, bool admin, string nombreUsuario, string nombre,
                       string apellidos, string correo, string imagen, string descripcion, DateTime edad,
                       int puntos, int puntosExperiencia, Nivel nivel)
        {
            UsuarioId = usuarioId;
            Contraseña = contraseña;
            Admin = admin;
            NombreUsuario = nombreUsuario;
            Nombre = nombre;
            Apellidos = apellidos;
            Correo = correo;
            Imagen = imagen;
            Descripcion = descripcion;
            Edad = edad;
            Puntos = puntos;
            PuntosExperiencia = puntosExperiencia;
            Nivel = nivel;
            Comunidades = new List<Comunidad>();
            Productos = new List<Producto>();
        }
    }